frontend_tests/node_tests/echo.js

```diff
@@ -29,6 +29,8 @@ const message_store = mock_esm("../../static/js/message_store", {
     get: () => ({failed_request: true}),
 
     update_booleans: () => {},
+
```

Comment:
I don't think this is super important to fix for your PR, but we generally want to get away from the pattern of having default implementations for functions in mocked dependencies unless they clearly apply to the whole file.  It's better to explicitly override them in each test.

Reply:
Yeah, agreed. Honestly I'm not sure there's a reason to mock `message_store` in this file at all.